theano/configdefaults.py

```diff
@@ -1647,7 +1649,7 @@ def short_platform(r=None, p=None):
 # Allow to have easily one compiledir per device.
 compiledir_format_dict['device'] = config.device
 compiledir_format_keys = ", ".join(sorted(compiledir_format_dict.keys()))
-default_compiledir_format = ("compiledir_%(short_platform)s-%(processor)s-"
+default_compiledir_format = ("compiledir_%(hostname)s-%(short_platform)s-%(processor)s-"
```

Comment:
I don't think it is a good idea. This mean that by default if your home is on nfs and you use Theano on multiple computers that are exactly the same, the cache won't be shared between those computers. This is what we did up to now.

What do you wanted to do with that change?